pattoo/db/schema/auth.py

```diff
@@ -0,0 +1,71 @@
+"""pattoo ORM Schema for User Authentication"""
```

Comment:
We have an authentication method used by `pattoo_mobile`. This looks like a duplication. Please review.